app/views/otherOfficials/AddSecondOtherOfficialsView.scala.html

```diff
@@ -0,0 +1,43 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.authorisedOfficials.routes._
+@import models.Mode
+@import views.ViewUtils._
+@import config.FrontendAppConfig
+
+@this(
+    govukLayout: templates.GovukLayoutWrapper,
+    h1: components.h1,
+    link: components.linkAsButton,
+    p: components.p
+)
+
+@(index: Index)(implicit request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
```

Comment:
Do we need index here?

Reply:
this will used for next iteration for second other offcial, as of now we are using dead end controller but apprerently it will be replaced by next page which i guess official Name
let me know if it sounds ok to you